test/Microsoft.VisualStudio.LanguageServices.Razor.Test/Xunit/ForegroundTheoryAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Microsoft.VisualStudio.LanguageServices.Razor.Test.Xunit
```

Comment:
Not sure why these changes are displayed here while the base branch is set to `rynowak/foreground-theory`

Reply:
github doesn't update the diff when you push to the base branch